cmd/main.go

```diff
@@ -32,6 +32,7 @@ import (
 	declarativev1 "sigs.k8s.io/kubebuilder/v3/pkg/plugins/golang/declarative/v1"
```

Comment:
I loved the gif showing how to use it. We might able to indeed use it in the plugins page that is amazing 🥇 

But how can we ensure that the UI/Graph is showing the metrics exported?
Did you test it out by following the quick start, have an Operator running with the metrics enabled in Prometheus and then check how the UI will show the data?
How does the UI itself render? Can we check an example to see how pretty is that?

Reply:
In the docs, we would have to explain in detail on what we need to de with the json file which we scaffold. You could link it to the grafana docs too, that explain on exporting the json to render the dashboard. https://grafana.com/docs/grafana/latest/dashboards/export-import/